test/server/admin/stats_params_test.cc

```diff
@@ -56,21 +56,11 @@ TEST(StatsParamsTest, ParseParamsFilter) {
   Buffer::OwnedImpl response;
 
   {
-    TestScopedRuntime scoped_runtime;
```

Comment:
since we removed this can we also remove the include on line 4?

Reply:
Done